projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -2990,11 +2975,19 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         return this._unpinnedWidth;
     }
 
+    /**
+     * @hidden
+     * Gets the combined width of the columns that are specific to the enabled grid features. They are fixed.
+     */
+    public get featureColumnsWidth() {
+        return this.getFeatureColumnsWidth();
```

Comment:
Why implement a method, which would be called inside a getter method? Also shouldn't some kind of caching be considered?

Reply:
This is because if I want to use in a child component that inherits the IgxGridBaseComponent `super.featureColumnsWidth` this is not supported for getters and in this case the child component only need to override the  'getFeatureColumnsWidth' method and it will be reflected in the getter as well.

Example is for the GroupBy in the IgxGridComponent that on top of the default calculations adds the expander column width. Also this is performed runtime and caching won't be viable here and the calculations are intensive anyway.

Reply:
This is available as a feature with the update to 8, but it's not in the current version.

Reply:
@skrustev Leave a TODO comment for this to be changed with 8.